staking/apr/compute.go

```diff
@@ -57,6 +56,57 @@ func expectedRewardPerYear(
 	return new(big.Int).Mul(expectedValue, oneYear), nil
 }
 
+func pastTwoEpochHeaders(
+	bc Reader,
+) (*block.Header, *block.Header, error) {
+	current := bc.CurrentHeader()
+	epochNow := current.Epoch()
+	oneEpochAgo, twoEpochAgo :=
+		new(big.Int).Sub(epochNow, common.Big2),
```

Comment:
sub big1.

Reply:
let's write unit tests for all new functions such as `pastTwoEpochHeaders`